lib/ts/recipe/session/claims.ts

```diff
@@ -0,0 +1,3 @@
+export { SessionClaim } from "./types";
```

Comment:
Adding validators through subclassing is fairly simple (see `StringClaim` in the `with-typescript/index.ts`), but I also added another proposal into the `supertokens-website` PR, where it can be done more dynamically.

Normally, built-in claim validators would be configured by parameters passed to the "getter" of the validator.

Validators added by other recipes could be configured by either:
1. Passing config from recipe init to the default validator.
2. The user disabling/removing the default claim and adding a properly configured one by themselves as a global validator.